internal/migrate/stage_contract.go

```diff
@@ -0,0 +1,100 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package migrate
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/contract-updater/lib/go/templates"
+	flowsdk "github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flowkit"
+	"github.com/onflow/flowkit/output"
+	"github.com/onflow/flowkit/transactions"
+	"github.com/spf13/cobra"
+
+	internaltx "github.com/onflow/flow-cli/internal/transactions"
+
+	"github.com/onflow/flow-cli/internal/command"
+)
+
+var stageContractflags struct{}
+
+var stageContractCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "stage-contract <CONTRACT_NAME>",
+		Short:   "stage a contract for migration",
+		Example: `flow migrate stage-contract HelloWorld`,
+		Args:    cobra.MinimumNArgs(1),
+	},
+	Flags: &stageContractflags,
+	RunS:  stageContract,
+}
+
+func stageContract(
```

Comment:
@jribbink

Reply:
> We don't have the --signer flag as stage/unstage contract will use the account holder as the signer/payer/authorizer. Would that not work?

I think it would work for most cases, but there are some cases where the same name contract is deployed across many different addresses - `SwapPair` in Increment's projects comes to mind. Would the current interface allow a project owner to easily distinguish which contract & account they want to stage in that case?

Reply:
do you think something along the lines of

```
flow migrate stage-contract ExampleNFT --account example-nft-account
```

would make sense? It'll use that account for signer/authorizer/payer all at once, but you would be able to specify which account from flow.json will be transacting on it.

Reply:
Yeah, I think that would work

Reply:
Quick confirmation - does the staging command replace import aliases according to the network being staged?

Reply:
currently it'll base it on `emulator`, but you can use the `network` flag to associate which network you want to use, if thats what youre asking

Reply:
`stage-contract` works with 

```
./main migrate stage-contract ExampleNFT --network testnet
```

But doesn't recognize the `--account` flag

Reply:
Also, the contract aliases aren't getting replaced meaning devs would have to statically declare their imports. So for example, I'd think that my CLI-supported aliased imports like
```
import "NonFungibleToken"
// or
import FungibleToken from "FungibleToken"
```

would be replaced with 
```
// on emulator it would be replaced with
import NonFungibleToken from 0xf8d6e0586b0a20c7
// or if I named with `--network testnet` it would be
import NonFungibleToken from 0x631e88ae7f1d7c20
```

But currently no aliases are updated in the process of staging

Reply:
let me take a look into this. I should bea ble to do it

Reply:
@sisyphusSmiling ive updated it now and it should work, could you test this with me?